frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -244,14 +244,19 @@ class PublicDashboardInner extends Component<PublicDashboardProps> {
         setParameterValueToDefault={setParameterValueToDefault}
         enableParameterRequiredBehavior
         actionButtons={
-          buttons.length > 0 && <div className={CS.flex}>{buttons}</div>
+          buttons.length > 0 ? <div className={CS.flex}>{buttons}</div> : null
```

Comment:
The previous type for `actionButtons` was `JSX.Element[]` which, as you can see in the `div`, was wrong